repos/system_upgrade/el7toel8/actors/tcpwrapperscheck/actor.py

```diff
@@ -0,0 +1,64 @@
+from leapp.actors import Actor
+from leapp.models import Report, TcpWrappersFacts, InstalledRedHatSignedRPM
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp.libraries.common.rpms import has_package
+from leapp.libraries.common.tcpwrappersutils import config_applies_to_daemon
+from leapp.libraries.common.reporting import report_with_links
+
+DAEMONS = [
+    ("audit", ["auditd"]),
+    ("bacula", ["bacula"]),
+    ("conserver", ["conserver"]),
+    ("cyrus-imapd", ["imap", "pop3", "imaps", "pop3s", "sieve", "lmtp"]),
+    ("dovecot", ["imap", "pop3", "imaps", "pop3s", "sieve", "lmtp"]),
+    ("nfs-utils", ["mountd", "statd"]),
+    ("openssh-server", ["sshd"]),
+    ("proftpd", ["proftpd"]),
+    ("quota", ["rquotad"]),
+    ("rpcbind", ["rpcbind"]),
+    # ("sendmail", ["sendmail"]),  # Handled in CheckSendmail Actor
+    ("slapi-nis", ["slapi-nis"]),
+    ("socat", ["socat"]),
+    ("stunnel", ["stunnel"]),
+    ("tftp-server", ["tftpd"]),
+    # ("vsftpd", ["vsftpd"]),  # Handled in VsftpdConfigCheck Actor
+    ("xinetd", ["xinetd"]),
+]
+
+
+class TcpWrappersCheck(Actor):
+    """
+    Check the list of packages previously compiled with TCP wrappers support
+    and check whether they have some rules configured in
+    /etc/hosts.{allow,deny} that are no longer honored by the daemons and
+    might make the service more accessible than expected.
+    """
+
+    name = 'tcp_wrappers_check'
+    consumes = (TcpWrappersFacts, InstalledRedHatSignedRPM,)
+    produces = (Report,)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        tcp_wrappers_facts = next(self.consume(TcpWrappersFacts))
```

Comment:
Please add an alert to the user if there are more messages than expected.
Something like [this](https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/actors/redhatsignedrpmcheck/libraries/library.py#L41)

Reply:
Thank you. I modified this also in the OpenSSH checks that I submitted some time before to do that.